rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -389,7 +385,21 @@ object Runtime {
         shortLeashParams,
         blockTime
       )
-    }).unsafeRunSync
+    })
+  }
+
+  // TODO: remove default store type
+  def create(
+      dataDir: Path,
+      mapSize: Long,
+      storeType: StoreType = LMDB,
+      extraSystemProcesses: Seq[SystemProcess.Definition[Task]] = Seq.empty
+  )(
+      implicit scheduler: Scheduler
+  ): Runtime[Task] = {
```

Comment:
what stops us from parameterizing this with F?

Reply:
Could we fix the call sites as well? (no longer return a bare Runtime - return F[Runtime] instead?)

Reply:
That is my intent. Though, if it proves too invasive I'll move this method to `src/test`